docs/guides/sdk/install.md

```diff
@@ -1,7 +1,7 @@
 # Install
 
 The fastest and easiest way to install the Cosmos SDK binaries
-is to run [this script](https://github.com/cosmos/cosmos-sdk/blob/develop/scripts/install.sh) on a fresh Ubuntu instance. Read the comments / instructions carefully (i.e., reset your terminal after running the script).
+is to run [this script](https://github.com/cosmos/cosmos-sdk/blob/develop/scripts/install_sdk_ubuntu.sh) on a fresh Ubuntu instance. Similarly, you can run [this script](https://github.com/cosmos/cosmos-sdk/blob/develop/scripts/install_sdk_bsd.sh) on a fresh FreeBSD instance. Read the comments / instructions carefully (i.e., reset your terminal after running the script).
```

Comment:
Hay, let's add a disclaimer to always read scripts before running them to make sure no untrusted connect is being downloaded - notably curl requests are always sketchy